pkg/kube/object_patch/types.go

```diff
@@ -0,0 +1,30 @@
+package object_patch
+
+type OperationSpec struct {
+	Operation   OperationType `json:"operation" yaml:"operation"`
```

Comment:
As we discussed internally, it is better to use `"sigs.k8s.io/yaml"`. And with `"sigs.k8s.io/yaml"` you only need to specify `json` tags.
https://play.golang.org/p/Vs4LQP3iL29

Reply:
We'll use yaml v3 with proper struct tags for each serialization format. More verbose, more correct.